C/plugins/storage/postgres/connection.cpp

```diff
@@ -443,6 +443,13 @@ SQLBuffer	jsonConstraints;	// Extra constraints to add to where clause
 								sql.append("\"");
 								sql.append((*itr)["column"].GetString());
 								sql.append("\"");
+								Logger::getLogger()->info("itr->HasMember(\"timezone\")=%s", itr->HasMember("timezone")?"True":"False");
```

Comment:
I would make this debug if you intend to keep it, certainly not a message that has meaning to a user